ddtrace/contrib/aiobotocore/patch.py

```diff
@@ -56,7 +56,7 @@ async def read(self, *args, **kwargs):
             # inherit parent attributes
             span.resource = self._self_parent_span.resource
             span.span_type = self._self_parent_span.span_type
-            span.meta = dict(self._self_parent_span.meta)
+            span._meta = dict(self._self_parent_span._meta)
```

Comment:
I could use Span.set_tag here but I held off due to this comment: https://github.com/DataDog/dd-trace-py/pull/3188#discussion_r796965314